queries.sql

```diff
@@ -1,3 +1,86 @@
-/*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+-- update the animals table by setting the species column to unspecified
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals WHERE species = 'unspecified';
+
+ROLLBACK;
+SELECT * FROM animals FROM species = 'unspecified';
+
+
+-- update the species field to be digimon and pokemon
+BEGIN TRANSACTION;
+UPDATE animals
+SET species = 'digimon' where name LIKE '%mon%';
+
+UPDATE animals 
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+SELECT * FROM animals;
+COMMIT;
+
```

Comment:
- [ ] Good job doing the query @BryteLitty, But please add a select statement after `COMMIT` to verify that the change persists after the commit.